content/sensu-go/5.16/guides/using-check-aggregates.md

```diff
@@ -0,0 +1,285 @@
+---
+title: ""
+linkTitle: "Using Check Aggregates"
+description: ""
+weight: 350
+version: "5.16"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.16:
+    parent: guides
+---
+
+Sensu provides the capability to monitor groups of checks or entities via aggregates. In Sensu Go, this is done through the use of labels. In this tutorial, we'll walk through configuring both an entity aggregate and a check aggregate.
+
+# Entity Aggregate
+
+Let's start with configuring an entity.
+
+The first thing we'll need to do is add a label to our agent.
+
+```yaml
+---
+# Sensu agent configuration
+
+##
+# agent overview
+##
+
+labels:
+  server_type: "webservers"
+
+```
+
+In this example, we've added the label `server_type` with the value `"webservers"`--it's useful to know if a number of agents on our webservers stop reporting in rather than just a single node. Let's add a similar label to a check.
```

Comment:
```suggestion
This example adds the label `server_type` with the value `"webservers"`. It's useful to know when a number of agents on your webservers stop reporting in rather than just a single node.

 Next, add a similar label to a check.
```

Reply:
With regard to "It's useful to know when a number of agents on your webservers stop reporting in rather than just a single node.", the connection between this point and adding the label "webservers" doesn't seem obvious.

A connecting point might be something like:
"...the value `"webservers"`. In practice, you would add this label to all of your webserver agents so that you would know when a number of agents on your webservers stop reporting in rather than just a single node."

I don't know if that's what you meant here...it may need further clarification (or maybe I missed the point entirely).